Update contributor-roadmap-participating.md (#537)
chaoss/community: how-to-contribute/contributor-roadmap-participating.md

# Contributor Roadmap - Participating

> There are several ways you can contribute to CHAOSS in different areas. 

Reviewer: ```suggestion
There are several ways you can contribute to CHAOSS. 
```

'in different areas' seemed the same as 'several ways', so I think this can be simplified?

> ## Contribution to CHAOSS Website

Reviewer: ```suggestion
## Contribution to the CHAOSS Website
```

> To contribute to the CHAOSS website, please join **wg-website** channel in the [Slack channel](https://join.slack.com/t/chaoss-workspace/shared_invite/zt-1fah5gu35-5oUQEPT32O2Zt~3MFVNMlw) where you can ask website related questions and provide your feedback. The website is hosted on Word Press, and most of the website content is pulled from [CHAOSS Website GitHub repo]( https://github.com/chaoss/website). Knowledge base content on the CHAOSS website is pulled from the CHAOSS Community GitHub repo]( https://github.com/chaoss/community). You can create issues and pull requests to contribute to GitHub; once approved, they will be reflected on the CHAOSS website.

Reviewer: ```suggestion
To contribute to the CHAOSS website, please join the **wg-website** channel in the [Slack channel](https://join.slack.com/t/chaoss-workspace/shared_invite/zt-1fah5gu35-5oUQEPT32O2Zt~3MFVNMlw) where you can ask questions and provide your feedback. The website is hosted on WordPress, and most of the website content is pulled from the [CHAOSS Website GitHub repo]( https://github.com/chaoss/website). Knowledge base content on the CHAOSS website is pulled from the CHAOSS Community GitHub repo]( https://github.com/chaoss/community). You can create issues and pull requests to contribute to GitHub; once approved, they will be reflected on the CHAOSS website.
```

## Contribution to CHAOSS Metrics and models

> CHAOSS metrics and models are developed in several [CHAOSS groups]( https://chaoss.community/kbtopic/chaoss-groups/). Each group has different meeting timings that can be found on [CHAOSS calendar]( https://chaoss.community/chaoss-calendar/), and each group has its [Slack channel](https://join.slack.com/t/chaoss-workspace/shared_invite/zt-1fah5gu35-5oUQEPT32O2Zt~3MFVNMlw) and GitHub repo where you can participate through discussion and by creating issues and pull request to make your contributions.

Reviewer: ```suggestion
CHAOSS metrics and models are developed in several [CHAOSS groups]( https://chaoss.community/kbtopic/chaoss-groups/). Each group has different meeting timings that can be found on the [CHAOSS calendar]( https://chaoss.community/chaoss-calendar/), and each group has a [Slack channel](https://join.slack.com/t/chaoss-workspace/shared_invite/zt-1fah5gu35-5oUQEPT32O2Zt~3MFVNMlw) and GitHub repo where you can participate through discussion and by creating issues and pull requests to make your contributions.
```

## Contribution to CHAOSS Software
CHAOSS has three [software projects]( https://chaoss.community/kb/software-projects/) implementing CHAOSS metrics and models. To contribute to any of the CHAOSS software, join their [Slack channel](https://join.slack.com/t/chaoss-workspace/shared_invite/zt-1fah5gu35-5oUQEPT32O2Zt~3MFVNMlw) to start a discussion. Each software project has its GitHub repo, where you can participate by creating issues and pull requests to make your contributions. 

> ## Contribution to CHAOSS Blogs

Reviewer: ```suggestion
## Contribution to the CHAOSS Blog
```

> To contribute to CHAOSS Blogs, join the [#wg-communications Slack channel](https://chaoss-workspace.slack.com/archives/C047TTUGGAJ)and share your idea on what you want to write. The Communications Working Group maintains a [spreadsheet of content ideas](https://docs.google.com/spreadsheets/d/1d4fCA5r3MUUxdlwTcB34V8gkicWFYU7hm_T2h74EHNM/edit#gid=1398016572) and a calendar of scheduled posts. You can find the list of topics [here.](https://docs.google.com/spreadsheets/d/1d4fCA5r3MUUxdlwTcB34V8gkicWFYU7hm_T2h74EHNM/edit#gid=1398016572) or suggest your topic. Once you have written your blog, share it with the community for feedback and publishing.

Reviewer: ```suggestion
To contribute to the CHAOSS Blog, join the [#wg-communications Slack channel](https://chaoss-workspace.slack.com/archives/C047TTUGGAJ)and share your idea on what you want to write. The Communications Working Group maintains a [spreadsheet of content ideas](https://docs.google.com/spreadsheets/d/1d4fCA5r3MUUxdlwTcB34V8gkicWFYU7hm_T2h74EHNM/edit#gid=1398016572) and a calendar of scheduled posts. You can find the list of topics [here.](https://docs.google.com/spreadsheets/d/1d4fCA5r3MUUxdlwTcB34V8gkicWFYU7hm_T2h74EHNM/edit#gid=1398016572) or suggest your topic. Once you have written your blog post, share it with the community for feedback and publishing.
```
